app/store/services/database.js

```diff
@@ -16,14 +16,14 @@ export const disconnect = () => {
   database.disconnect();
 };
 
-export const registerCallBackForEvent = cb => {
-  database.registerForEvent(cb);
+export const watchData = cb => {
```

Comment:
name should be onChange and it should return a subscription and this subscription should allow you to stop watch